modules/installation-configuration-parameters.adoc

```diff
@@ -2008,10 +2056,6 @@ Additional Azure configuration parameters are described in the following table:
 |Defines the azure instance type for compute machines.
 |String
 
-|`compute.platform.azure.zones`
-|The availability zones where the installation program creates compute machines.
-|String list
-
```

Comment:
@xenolinux In additional azure stack hub parameters table,  parameter diskType is not described accurately, since the disk type on ASH supports `standard_LRS`, `premium_LRS`.
So need to remove `standardSSD_LRS` from `compute.platform.azure.osDisk.diskType`, `controlPlane.platform.azure.osDisk.diskType` and `platform.azure.defaultMachinePlatform.osDisk.diskType`.

Same [comments](https://github.com/openshift/openshift-docs/pull/60073/files#r1196075223) as Azure, also need to remove `standard_LRS` from `platform.azure.defaultMachinePlatform.osDisk.diskType` in ASH additional parameter table.

Reply:
please ignore this comments as it is addressed by doc bug: https://issues.redhat.com/browse/OCPBUGS-13804